components/function-runtimes/nodejs12/Dockerfile

```diff
@@ -1,4 +1,128 @@
-FROM node:12.21.0-alpine3.12
+# Node 12 Alpine image from https://github.com/nodejs/docker-node/tree/main/12/alpine3.12
```

Comment:
I would add such comment:
`We use this code to manually use newest alpine and nodejs. Use official build when it will be available`

The same comment should be in nodejs10 and Python.